examples/HttpCodesApi/GetCode.php

```diff
@@ -18,25 +18,25 @@ public static function handle(
         stdClass $args,
         callable $next
     ): ResponseInterface {
-        foreach (self::getCodes() as $code) {
-            if ($args->code === $code->code) {
-                if ($args->type === "json") {
-                    return new JsonResponse($code);
-                }
-
-                ob_start();
-                require __DIR__ . "/HtmlResponse.php";
-                $html = ob_get_clean();
-
-                return new HtmlResponse($html);
-            }
+        $codes = self::getCodes();
+
+        if (!($code = $codes[$args->code])) {
+            return throw new HttpNotFound($request);
+        }
+
+        if ($args->type === "json") {
+            return new JsonResponse($code);
         }
 
-        return throw new HttpNotFound($request);
+        ob_start();
+        require_once __DIR__ . "/HtmlResponse.php";
+        $html = ob_get_clean();
+
+        return new HtmlResponse($html);
     }
 
-    public static function getCodes()
+    public static function getCodes(): array
     {
-        return json_decode(file_get_contents(__DIR__ . "/../../tools/HttpExceptions/HttpCodes.json"));
+        return json_decode(file_get_contents(__DIR__ . "/../../tools/HttpExceptions/HttpCodes.json"), true);
```

Comment:
Still using json instead of php to store the data

Reply:
Well this is just an example so I wouldn't worry too much about it 😅

Reply:
youre also using this json in other places, no?

Reply:
Just for generator stuff nothing else